tests/webview/test_google_redirect.py

```diff
@@ -0,0 +1,18 @@
+from requests import get
+from tests import markers
+
+
+@markers.webview
+@markers.nondestructive
+def test_google_redirect_archive_results(webview_base_url, chrome_options):
```

Comment:
the items (fixtures) passed into this function are not being used. In your test function. I believe you should remove what is not needed. From what i can tell it's `chrome_options`

Reply:
Sorry, I missed it because I _was_ using them before.